### DISCUSSION TITLE
How to integrate automation scripts

### DISCUSSION THREAD

**Discussion by Hegelim** [gnzsnz/ib-gateway-docker#146] [Q&A] [answered]
Read the instructions but haven't been able to figure this out yet. Suppose I have some auto trading scripts written in Python, how to integrate my scripts with this docker image?

**Comment by gnzsnz** [comment, marked as answer]
here you have an example of a docker-compose with a separate image running a trading strategy

https://github.com/gnzsnz/ib-gateway-docker/discussions/124 and here https://github.com/gnzsnz/ib-gateway-docker/discussions/140
